en/flight_modes_fw/altitude.md

```diff
@@ -28,11 +37,15 @@ RC/manual mode like Stabilized mode but with altitude stabilization (centered st
     This is same as in [Stabilized mode](../flight_modes_fw/stabilized.md).
   - Yaw stick actuates the rudder (signal will be added to the one calculated by the autopilot to maintain [coordinated flight](https://en.wikipedia.org/wiki/Coordinated_flight)).
     This is same as in [Stabilized mode](../flight_modes_fw/stabilized.md).
-
-:::note
-
-- Manual input is required (RC controller, or gamepad/thumbsticks through MAVLink).
-- The altitude is normally measured using a barometer, which may become inaccurate in extreme weather conditions.
+- Manual control input is required (such as RC control, joystick).
+  - Roll: Assistance from autopilot to hold altitude against wind.
+  - Pitch: Assistance from autopilot to stabilize the attitude.
+    Position of RC stick maps to the orientation of vehicle.
+  - Yaw: Input is sent directly to control allocation.
+  - Throttle: Assistance from autopilot to hold altitude against wind.
```

Comment:
FYI @sfuhrer I'm trying to get rid of the tables in https://docs.px4.io/main/en/flight_modes/#fixed-wing - its HTML and not all that easy to "parse" as a reader. 

So the entry points to each mode is now a high level topic like we previously used for the getting started flight modes topic.

But I didn't want to throw away the information in the table for what is controlled in each axis, so I have pulled it here into the technical summary. Hope this makes sense to you.

![image](https://github.com/PX4/PX4-user_guide/assets/5368500/4638fcbd-10d0-497f-9337-37b918af442a)

Reply:
For me it would be okay to drop that table, I never found it very readable. Words are better here in my eyes.